src/Jackett.Common/Indexers/EliteTracker.cs

```diff
@@ -22,6 +23,7 @@ private string LoginUrl
         { get { return SiteLink + "takelogin.php"; } }
         private string BrowseUrl
         { get { return SiteLink + "browse.php"; } }
+        private bool TorrentHTTPSMode => ((ConfigurationDataEliteTracker)configData).TorrentHTTPSMode.Value;
 
         private new ConfigurationDataBasicLogin configData
```

Comment:
Change this and line 30 to ConfigurationDataEliteTracker, then you shouldn't need the case in line 26.